css-layout-classes/styles.css

```diff
@@ -0,0 +1,42 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.row {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.column-third {
+  flex: 1; /* Use flex: 1 to allow columns to occupy equal space */
+}
```

Comment:
I would use `flex-basis` rather than using the `flex` shorthand. `flex-basis: calc(100% / 3)` or `flex-basis: 33.3%`